pandas/conftest.py

```diff
@@ -271,7 +271,7 @@ def axis(request):
     return request.param
 
 
-@pytest.fixture(params=[True, False, None])
+@pytest.fixture(params=[True, False])
```

Comment:
Stumbled into this because of the added test. This should have been removed by #57330